cluster/plugin/aws/plugin.go

```diff
@@ -61,7 +63,7 @@ func CreateStack(ctx context.Context, svc *cf.CloudFormation, opts *RequestOptio
 		},
 		OnFailure:        aws.String(opts.OnFailure),
 		Parameters:       toParameters(opts.Params),
-		TemplateURL:      aws.String(templateURL),
+		TemplateURL:      aws.String(DefaultTemplateURL),
```

Comment:
Yup to both ... I did a search/replace on templateURL to rename, but didn't switch these over to use the option now. Thanks for catching.